packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2309,6 +2388,13 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     widget.onAppPrivateCommand?.call(action, data);
   }
 
+  @override
+  void commitContent(Map<String, dynamic> content) {
```

Comment:
consider converting the `Map` to `CommitContent` in the services layer. It's easier to work with typed data.

Reply:
Done